artefacts/README.md

```diff
@@ -1 +1,16 @@
+[<< Back](https://cntt-n.github.io/CNTT/)
+# Handling of figures
 
+## Table of Contents
+
+* [1 Usage of figures](#1)
+
+<a name="1"></a>
+## 1 Usage of figures
+
+- Figures used in the document should be in Portable Network Graphic (PNG) format
+- Figures must be stored in the `figures` folder of the document where the figures are used (e.g.: `doc/ref_arch/figures/`)
+- File name of the figures shold contain the chapter number where the figure is used as a prefix
+- Every figure used in the documents should have an editable version commited to the repository in `ppt(x)` or in `svg` format
+- Editable versions should be stored in the `figures/artefacts` folder of the document where the figures are used (e.g.: `doc/ref_arch/figures/artefacts`)
```

Comment:
Can we keep the artefacts directory at the same level as the figures? Artefacts may include items other than the figures.

Reply:
Okay for me, but please check @walterkozlowski -s comment and my question on the topic.

Reply:
My suggestion as a compromise (reg the location):
Figures must be stored in the `figures` folder of the document where the figures are used ( `doc/ws_name/figures/`)
Editable versions should be stored in the `figures/editable` folder of the document where the figures are used  `doc/ws_name/figures/editable`)
This will provide a clear direction where the editable versions are stored.
@CsatariGergely, it is clear how the above would work for editable ppt and display png files.  It seems some confusion what is editable and display version for svg format, see @kedmison 's comments.  If this can be clarified, then i do not have anything against svg format.  Can you clarify please?